### PR TITLE
chore: add status information to workspaces

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: workspaces/getting-started.md [hunk 1]
```diff
@@ -31,9 +31,30 @@ Coder redirects you to an overview page for your workspace during the build
...
+### Workspace status
+
+The workspace overview page displays information regarding the status and
+performance of your workspace.
+
+![Workspace overview](../assets/workspaces/status-chip.png)
+
+The following workspace statuses are available:
+
+- **Running**: Your workspace is running
```
Comment (jawnsy) [inline workspaces/getting-started.md:49]: I think a diagram showing the different states would be useful here, or in the linked lifecycle page (showing the different actions and possible state transitions)

Comment (khorne3) [reply workspaces/getting-started.md:49]: @jawnsy That's a good idea; I'll request that one be made

Comment (khorne3) [reply workspaces/getting-started.md:49]: update: working w/ Jeb re: diagrams

Comment (jawnsy) [reply workspaces/getting-started.md:49]: It's useful even without diagrams for sure! Was just suggesting that we add a state chart to show these states and the possible transitions. I'm not sure the best person to review it though.

I'll share a quick example of what I mean 🙂

**Comment by ericpaulsen** [comment]
What's the current status for this PR? Are we still waiting on a diagram design from Jeb?

**Comment by khorne3** [comment]
@ericpaulsen We have diagrams WIP (I have a separate CH card tracking that) but I'd like to get this info out at least

**Review by ericpaulsen — APPROVED** [review]

**Pull request merged**